client/config/config_test.go

```diff
@@ -74,10 +72,7 @@ func TestConfigCmdEnvFlag(t *testing.T) {
 		args    []string
 		expNode string
 	}{
-		{"env var is set with no flag", testNode1, []string{"validators"}, testNode1},
-		{"env var is set with a flag", testNode1, []string{"validators", fmt.Sprintf("--%s=%s", flags.FlagNode, testNode2)}, testNode2},
 		{"env var is not set with no flag", "", []string{"validators"}, defaultNode},
-		{"env var is not set with a flag", "", []string{"validators", fmt.Sprintf("--%s=%s", flags.FlagNode, testNode2)}, testNode2},
```

Comment:
Did these break or something?

Reply:
My bad. I was debugging and accidentally committed the temp change. Will revert